packages/client/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
We can remove the icon as well.
Don't we have an Uncharted one while waiting to make a TERArium icon?